.golangci.yml

```diff
@@ -8,7 +8,6 @@ linters-settings:
   goimports:
     local-prefixes: github.com/mattermost/mattermost-starter-template
   govet:
-    check-shadowing: true
```

Comment:
@wetneb shouldn't this get replaced with `shadow`?  Otherwise shadow var checks are disabled.  Or am I missing something?

Reply:
The warning message says:

> The configuration option `linters.govet.check-shadowing` is deprecated. Please enable `shadow` instead, if you are not using `enable-all`.

Because we are already using `enable-all`, I believe `shadow` is already enabled and so does not need explicit enabling.